src/com/company/InventoryManager.java

```diff
@@ -0,0 +1,32 @@
+package com.company;
+
+import java.util.HashMap;
+
+public class InventoryManager implements IInventory {
+    HashMap<ItemType,ItemStack> inventory = new HashMap<>();
+    @Override
+    public void addItem(ItemType itemType, int count) throws Exception {
+
+        if(inventory.get(itemType)==null) inventory.put(itemType, new ItemStack(itemType));
```

Comment:
Spacinig here and lack of {} makes this harder to read. Typically its suggested to not commit {} unless its very trivial.